<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>674</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>测试</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>预测结果</string>
          </property>
          <widget class="QTextEdit" name="textEdit">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>60</y>
             <width>621</width>
             <height>41</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>模型路径</string>
          </property>
          <widget class="QLineEdit" name="lineEdit_4">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>50</y>
             <width>831</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>C:\Users\ASUS\Desktop\faceqimo\Mat</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="maximumSize">
           <size>
            <width>1000</width>
            <height>150</height>
           </size>
          </property>
          <property name="title">
           <string>多张图片路径选择</string>
          </property>
          <widget class="QLineEdit" name="lineEdit_3">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>831</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>C:\Users\ASUS\Desktop\faceqimo\pic\test</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_3">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>80</y>
             <width>831</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>组预测</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>单张图片路径选择</string>
          </property>
          <widget class="QLineEdit" name="lineEdit_5">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>831</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>C:\Users\ASUS\Desktop\faceqimo\pic\test\s1\t (1).jpeg</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_4">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>80</y>
             <width>831</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>单预测</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>训练</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_7">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <property name="title">
           <string>训练情况</string>
          </property>
          <widget class="QTextEdit" name="textEdit_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>861</width>
             <height>51</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>数据集路径</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0">
            <widget class="QLineEdit" name="lineEdit">
             <property name="text">
              <string>C:\Users\ASUS\Desktop\faceqimo\pic\train</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>模型保存路径</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLineEdit" name="lineEdit_2">
             <property name="text">
              <string>C:\Users\ASUS\Desktop\faceqimo\Mat</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>开始训练</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>930</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>可爱鸭鸭略</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="ya.qrc"/>
 </resources>
 <connections/>
</ui>
